prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -774,6 +774,11 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             .execute()
     }
 
+    fun refreshMaterializedViews(tableName: String, txn: DataAccessTransaction? = null) {
+        val ctx = if (txn != null) DSL.using(txn) else create
+        Routines.refreshMaterializedViews(ctx.configuration(), tableName)
+    }
+
```

Comment:
Created a proc that takes the table name and refreshes the materialized view based on the table name.